### PR TITLE
Update inlines.py

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: plugins/inlines.py [hunk 1]
```diff
@@ -82,3 +83,34 @@ async def html_inline(c: Client, q: InlineQuery):
...
+                    ),
+                )
+            ]
+        )
+    await q.answer(
+        [
+            InlineQueryResultArticle(
+                title="click here to get the information about the user",
+                input_message_content=InputTextMessageContent(
+                    f"username: {user.username} \n id: {user.id} \n dc: {user.dc_id} \n link to the user: {user.mention()}",
```
Comment (alissonlauffer) [inline plugins/inlines.py:112]: Why do you add spaces before `\n`?

Comment (iiiiii1wepfj) [reply plugins/inlines.py:112]: which space?

Comment (alissonlauffer) [reply plugins/inlines.py:112]:
```suggestion
                    f"username: {user.username}\nid: {user.id}\ndc: {user.dc_id}\nlink to the user: {user.mention()}",
```

**Comment by iiiiii1wepfj** [comment]
done

**Pull request merged**